apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx

```diff
@@ -0,0 +1,104 @@
+
+## What is new in v1?
+
+supabase-flutter v1 focuses on improved developer experience. The new version requires far less boiler plate code as well as it provides more intuitive APIs. Here are some highlights of the update.
+
+### No more `.execute()`
+
+Previously, for `.select()`, `.insert()`, `.update()`, `.delete()` and `.stream()` required `execute()` to be called at the end. On top of that, errors are thrown, not returned, so you can be sure that you have the query results in the returned value.
+
+```dart
+// Before
+final response = await supabase.from('messages').select().execute();
+final data = response.data;
+
+// After
+final data = await supabase.from('messages').select();
+```
+
+### More predictable auth methods
+
+Names of the auth methods are more descriptive about what they do. Here are some examples of the new methods:
+
+```dart
+await supabase.auth.signInWithPassword(email: email, password: password);
+
+await supabase.auth.signInWithOAuth(Provider.github)
+```
+
+Also `onAuthStateChange` returns stream, which feels more natural for anyone coding in Dart.
+
+```dart
+supabase.auth.onAuthStateChange.listen((data) {
+  final AuthChangeEvent event = data.event;
+  final Session? session = data.session;
+});
+```
+
+### Realtime Multiplayer edition support
+
+Supabase introduced realtime multiplayer edition two launch weeks ago. supabase-flutter now has first class support for the two newly introduced realtime methods, broadcast and presence. Broadcast can be used to share realtime data to all connected clients with low latency. Presence is a way to let other connected clients know the status of the client. You can visit [multiplayer.dev](http://multiplayer.dev) to see a quick demo of the feature.
+
+```dart
+final channel = Supabase.instance.client.channel('my_channel');
+
+// listen to `location` broadcast events
+channel.on(
+    RealtimeListenTypes.broadcast,
+    ChannelFilter(
+      event: 'location',
+    ), (payload, [ref]) {
+	// Do something exciting with the broadcast event
+});
+
+// send `location` broadcast events
+channel.send(
+  type: RealtimeListenTypes.broadcast,
+  event: 'location',
+  payload: {'lat': 1.3521, 'lng': 103.8198},
+);
+
+// listen to presence states
+channel.on(RealtimeListenTypes.presence, ChannelFilter(event: 'sync'),
+    (payload, [ref]) {
+	// Do something exciting with the presence state
+});
+
+// subscribe to the above changes
+channel.subscribe((status) async {
+  if (status == 'SUBSCRIBED') {
+    // if subscribed successfully, send presence event
+    final status = await channel.track({'user_id': myUserId});
+  }
+});
+```
+
+These are just tip of the iceburg of all the updates that we shipped in v1. Check out the [documentation](https://supabase.com/docs/reference/dart/) to see the full list of what updates supabase-flutter can do.
+
+## Acknoledgements
```

Comment:
minor typo - Acknoledgements instead of Acknowledgements

Reply:
Thanks for catching this one!
```suggestion
## Acknowledgements
```